### PR TITLE
ISSUE #3392 - Edit Federation Modal uses default horizontal scrollbar, which is only visible at bottom of vertical scroll

### LINKED ISSUE #3392: Edit Federation Modal uses default horizontal scrollbar, which is only visible at bottom of vertical scroll
### Description

If you open the Edit Federation modal and then reduce the screen size a horizontal scrollbar will appear, however it is only visible when the user scrolls to the bottom.

This is happening on Firefox, Edge, and Chrome. On Edge and Chrome the horizontal scrollbar is the default browser scrollbar, and not the custom react one.

- This is happening in: staging
- This requires a hotfix to production: no 

### Steps to replicate
- Open the Edit Federation modal in a project with lots of containers
- Reduce the screensize until the search bar starts to overflow its container
- Scroll down and you will see a horizontal scrollbar

### Current Behaviour
- The search fields overflow causing a horizontal scroll bar to appear
- This scrollbar is only visible when the user is scrolled to the bottom of the modal

### Expected Behaviour
- Either the horizontal scrollbar doesn't need to appear, or it is visible at any vertical scroll level.

### Pictures/ Gifs of the error

image
Note the lack of horizontal scroll bar. In this picture I have scrolled right and then up which obscures part of the list.

### PR DESCRIPTION
**Pull request by The-Daniel**
This fixes #3392 

#### Description
- Custom horizontal scrollbar is not visible on edit federations modal
- accordion elements scale better with screen size
- The accordion title is still 1 line, as using multilines and truncating the title and not the container number was a difficulty

#### Test cases
Open up the edit federation (try using both short and long federation names), and reduce the screen width

### CODE REVIEW

**Comment by Amantini1997** [comment]
I know it's not something you changed in this branch, but when shrunk, the searchbar adds a padding-left which looks odd 
image

File: frontend/src/v5/ui/controls/modal/formModal/formDialog.component.tsx [hunk 1]
```diff
@@ -47,6 +47,7 @@ export interface IFormModal extends Omit<DetailedHTMLProps<FormHTMLAttributes<HT
 	isSubmitting?: boolean;
 	disableClosing?: boolean;
 	hideSubmitButton?: boolean;
+	hideHorizontalScroll?: boolean;
```
Comment (Amantini1997) [inline frontend/src/v5/ui/controls/modal/formModal/formDialog.component.tsx:50]: I am not sure Santiago will like this 🤣

**Review by Amantini1997 — DISMISSED** [review]

File: frontend/src/v5/ui/controls/modal/formModal/formDialog.component.tsx (hunk 1 above)
Comment (The-Daniel) [reply frontend/src/v5/ui/controls/modal/formModal/formDialog.component.tsx:50]: Shhhhh

**Review by george-hadfield — APPROVED** [review]

**Review by Amantini1997 — APPROVED** [review]

**Comment by george-hadfield** [comment]
Approved.

**Review by george-hadfield — APPROVED** [review]

**Pull request merged**